utils/email_service.py

```diff
@@ -1,17 +1,55 @@
 from celery import shared_task
 from decouple import config
-from django.core.mail import EmailMessage
+from django.core.mail import EmailMessage, send_mail
 from decouple import config
+import boto3
 
 
 @shared_task
-def send_email(recipient,subject,body):
-    if config('MOCK_EMAIL', cast=bool, default=True):
+def send_email(recipient, subject, body):
+    if config('MOCK_EMAIL', cast=bool):
         print('Mock email service:')
-        print('Subject:',subject)
-        print('Recipients:',', '.join(recipient))
-        print('Subject:',subject)
+        print('Subject:', subject)
+        print('Recipients:', ', '.join(recipient))
+        print('Subject:', subject)
     else:
-        msg = EmailMessage(subject, body, config('EMAIL_HOST_USER') , recipient)
-        msg.content_subtype = "html"  
-        msg.send()
\ No newline at end of file
+        if config('EMAIL_BACKEND', cast=str) == 'django_amazon_ses.EmailBackend':
+            client = boto3.client(
+                'ses',
+                aws_access_key_id=config('AWS_ACCESS_KEY_ID'),
+                aws_secret_access_key=config('AWS_SECRET_ACCESS_KEY'),
+                region_name='eu-west-1'
+            )
+            try:
+                sender_email = config('EMAIL_HOST_USER')
+                print(sender_email)

```

Comment:
This may not serve any purpose here I guess, you can remove this print statement.